v3/src/components/case-card/case-card.v2.test.tsx

```diff
@@ -10,6 +10,7 @@ import "./case-card.v2"
 const { CaseCard } = DG.React as any
 
 describe("CaseCard component", () => {
+  jest.setTimeout(10000)
```

Comment:
Is this just adding a 10 second wait to our jest tests? 😢

Reply:
It's raising the failure timeout, so if it succeeds in less time it will proceed without waiting. It just means that it will wait 10 sec (rather than 5) before deciding to fail the test based on lack of response. While testing locally, I had it fail the 5-sec timeout a few times so I thought it prudent to extend it. In other words, it's not a `wait` in the cypress sense.